reporter/reporter.go

```diff
@@ -97,7 +97,12 @@ func (r *reporter) ReportCounter(id id.ID, value int64) error {
 	multiErr := xerrors.NewMultiError()
 	matchRes := r.matcher.Match(id)
 	earliestNanos := r.nowFn().Add(-r.maxNegativeSkew).UnixNano()
-	if err := r.server.WriteCounterWithPoliciesList(id.Bytes(), value, matchRes.MappingsAt(earliestNanos)); err != nil {
+	if err := r.server.WriteCounterWithPoliciesList(
+		id.Bytes(),
+		value,
+		// NB(cw) We do not support custom aggregation types for Counter in mapping rules.
```

Comment:
This is to make the mapping rules more flexible, or we could enforce a type tag in the mapping rules when user want to include aggregation types. 

On a second thought, that's probably fine actually, I'm open to remove this and move the check to UI.